deps.ts

```diff
@@ -1,6 +1,10 @@
-export * as dotenv from "https://deno.land/std@0.203.0/dotenv/mod.ts";
-export * as discord from "https://deno.land/x/discord_api_types@0.37.60/v10.ts";
+export * as dotenv from "https://deno.land/std@0.208.0/dotenv/mod.ts";
+export * as discord from "https://deno.land/x/discord_api_types@0.37.65/v10.ts";
 export * from "https://deno.land/x/codemod@0.0.5/github/mod.ts";
 export type { GitHubAPIClientOptions } from "https://deno.land/x/codemod@0.0.5/github/api/mod.ts";
 export { Duration } from "https://deno.land/x/durationjs@v4.1.0/mod.ts";
 export { default as nacl } from "https://esm.sh/tweetnacl@1.0.3";
+export {
+  type AppSchema,
+  createApp,
+} from "https://etok.codes/discord_app/raw/0f6dfc18897ecbd77452d2e5753e1c6da993f526/mod.ts";
```

Comment:
Replace with `deno.land/x` import URL.

Reply:
Resolved in <https://github.com/acmcsufoss/shorter/pull/9/commits/078ea4b813a4f84fae9089d4e212f94242d63750>.